mandatory/3-playing-computer.js

```diff
@@ -18,17 +18,18 @@
 let x = 2;
 let a = 6;
 
+
 const f1 = function (a, b) {
-  return a + b;
+  return a;
 };
 
 const f2 = function (a, b) {
-  return a + b + x;
+  return a + x;
 };
 
 console.log(x);
 console.log(a);
-console.log(b);
+// console.log(b);
```

Comment:
Was expecting to see the responses to the questions asked in this exercise. Please provide your answers